test/e2e.sh

```diff
@@ -181,7 +181,7 @@ main() {
 
     echo "Add git remote k8s ${CHARTS_REPO}"
     git remote add storageos "${CHARTS_REPO}" &> /dev/null || true
-    git fetch storageos master
+    git fetch --prune --unshallow storageos master
```

Comment:
Maybe a comment about why prune unshallow? Maybe just here on github PR review if not as code comment.

Reply:
I don't think it's needed, removing it.